media-gfx/apng2gif/apng2gif-1.7.recipe

```diff
@@ -0,0 +1,46 @@
+SUMMARY="Converts APNG animations into GIF format"
+DESCRIPTION="This program converts APNG animations into animated GIF format. \
+It performs the reverse of gif2apng."
+HOMEPAGE="http://apng2gif.sourceforge.net/"
+COPYRIGHT="2009-2015 Max Stepin"
+LICENSE="Zlib"
+REVISION="1"
+SOURCE_URI="https://downloads.sourceforge.net/apng2gif/apng2gif-$portVersion-src.zip"
+CHECKSUM_SHA256="6c7b2cce851378f2fc8f6b490ce098e9cf0038aa05da7099771c8be94620e6ca"
+SOURCE_DIR=""
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
```

Comment:
OK. fine.
Let me update the GCI task with a "Request More Work", otherwise we might forget and Google will think Haiku is not responding quickly enough ;-)